### PR TITLE
[GE-117] Add support for adding ssh key

### PR DESCRIPTION
**Pull request by octosteve**

### CODE REVIEW

File: lib/learn_web/client/ssh_keys.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'learn_web/client/ssh_keys/add'
+
+module LearnWeb
+  class Client
+    module SshKeys
+      def add_ssh_key_endpoint
+        "#{API_ROOT}/ssh_keys"
+      end
+
+      def add_ssh_key(key:, key_title: nil, client: nil)
```
Comment (octosteve) [inline lib/learn_web/client/ssh_keys.rb:10]: Public method to add key and key_title.

Comment (octosteve) [reply lib/learn_web/client/ssh_keys.rb:10]: Allows for injecting client on this method only. Used for testing.

File: Guardfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+guard :rspec, cmd: "bundle exec rspec" do
```
Comment (ktravers) [inline Guardfile:1]: better late than never

**Review by ktravers — APPROVED** [review]
TESTS 🎉

**Review by talum — COMMENTED** [review]
I was about to approve this. But anyway 👍 ⚡️ 🎖 🏅 🥇 🥈 🥉 🍖

**Review by talum — COMMENTED** [review]
I think we still need a couple Learn API changes to support the multiple accounts fully though.

**Pull request merged**